bot/help.py

```diff
@@ -0,0 +1,571 @@
+
+                cat_cmds = []
+
+                # format details for each child command
+                for command in cmds:
+
+                    # skip if hidden and hide if session is set to
+                    if command.hidden and not self._show_hidden:
+                        continue
+
+                    # see if the user can run the command
+                    strikeout = ''
+
+                    # Patch to make the !help command work outside of #bot-commands again
+                    # This probably needs a proper rewrite, but this will make it work in
+                    # the mean time.
+                    try:
+                        can_run = await command.can_run(self._ctx)
+                    except CheckFailure:
+                        can_run = False
+
+                    if not can_run:
+                        # skip if we don't show commands they can't run
+                        if self._only_can_run:
+                            continue
+                        strikeout = '~~'
+
+                    signature = self._get_command_params(command)
+                    info = f"{strikeout}**`{prefix}{signature}`**{strikeout}"
+
+                    # handle if the command has no docstring
+                    if command.short_doc:
+                        cat_cmds.append(f'{info}\n*{command.short_doc}*')
+                    else:
+                        cat_cmds.append(f'{info}\n*No details provided.*')
+
+                # state var for if the category should be added next
+                print_cat = 1
+                new_page = True
+
+                for details in cat_cmds:
+
+                    # keep details together, paginating early if it won't fit
+                    lines_adding = len(details.split('\n')) + print_cat
+                    if paginator._linecount + lines_adding > self._max_lines:
+                        paginator._linecount = 0
+                        new_page = True
+                        paginator.close_page()
+
+                        # new page so print category title again
+                        print_cat = 1
+
+                    if print_cat:
+                        if new_page:
+                            paginator.add_line('')
+                        paginator.add_line(category)
+                        print_cat = 0
+
+                    paginator.add_line(details)
+
+        # save organised pages to session
+        self._pages = paginator.pages
+
+    def embed_page(self, page_number: int = 0) -> Embed:
+        """Returns an Embed with the requested page formatted within."""
+        embed = Embed()
+
+        # if command or cog, add query to title for pages other than first
+        if isinstance(self.query, (commands.Command, Cog)) and page_number > 0:
+            title = f'Command Help | "{self.query.name}"'
+        else:
+            title = self.title
+
+        # Original: embed.set_author(name=title, icon_url=constants.Icons.questionmark)
```

Comment:
The constant from the bot is a [Discord provided emoji](https://github.com/python-discord/bot/blob/master/config-default.yml#L101), how about adding it to Seasonalbot's constants rather than removing it completely?